aml/org.testeditor.aml.model/src/org/testeditor/aml/ModelUtil.xtend

```diff
@@ -47,7 +47,10 @@ class ModelUtil {
 	 * @return all interaction types of the {@link Component} as well as its elements.
 	 */
 	def Set<InteractionType> getAllInteractionTypes(Component component) {
-		return (component.componentInteractionTypes + component.componentElementsInteractionTypes).toSet
+		if(component != null){
```

Comment:
formatting is broken, also please use `!==` for null comparisons (`!=` is the equals operator)